src/lib/Hydra/Controller/Jobset.pm

```diff
@@ -270,8 +270,8 @@ sub updateJobset {
     $jobset->jobsetinputs->delete;
 
     if ($type == 0) {
-        foreach my $name (keys %{$c->stash->{params}->{jobsetinputs}}) {
-            my $inputData = $c->stash->{params}->{jobsetinputs}->{$name};
+        foreach my $name (keys %{$c->stash->{params}->{inputs}}) {
+            my $inputData = $c->stash->{params}->{inputs}->{$name};
```

Comment:
This JS function will need to be updated to account for this change: https://github.com/NixOS/hydra/blob/master/src/root/edit-jobset.tt#L223-L243